src/features/content-answers/__tests__/ContentAnswersModalContent.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import ContentAnswersModalContent from '../ContentAnswersModalContent';
+
+describe('features/content-answers/ContentAnswersModalContent', () => {
+    const renderComponent = (props?: {}) => {
+        render(<ContentAnswersModalContent fileName="" {...props} />);
```

Comment:
should we test the differences between fileName being an empty string vs undefined vs a string with length?

Reply:
its required, adding a name